.github/workflows/GHPages.yml

```
@@ -28,12 +28,12 @@ jobs:
       - uses: actions/setup-node@v3
```

I think `pnpm/action-setup` is necessary.

```suggestion
      - uses: pnpm/action-setup@v2
      - uses: actions/setup-node@v3
```

---

I couldn't do this perfectly😇
fixed!
https://github.com/sveltejs/svelte-eslint-parser/pull/334/commits/92a644bd23fced4017c9baea3bfd12a72d3587f1